orttraining/orttraining/python/training/artifacts.py

```diff
@@ -105,6 +107,10 @@ def build(self, *inputs_to_loss):
         for arg in requires_grad:
             training_block.requires_grad(arg)
 
+    if frozen_params is not None:
+        for arg in frozen_params:
+            training_block.requires_grad(arg, False)
+
```

Comment:
nit: maybe we should add a check when in frozen_prams loops in case users pass a weight name in both requires_grad and frozen_params.

Reply:
There is a check inside training Blick. So it should be OK for user to add to both.

Reply:
But I think having a check here makes sense. Will add it.